pages/Salesforce.html

```diff
@@ -29,12 +29,14 @@
     <div class="coveo-main-section">
       <div class="coveo-facet-column">
         <div class="CoveoFacet" data-title="Type" data-field="@objecttype" data-number-of-values="5" data-tab="Salesforce"></div>
+        <div class="CoveoFacet" data-title="Status" data-field="@sfstatus" data-number-of-values="5" data-tab="Salesforce"></div>
```

Comment:
I would remove the number of values on both facet, since 5 is already the default.